minichain/p2p.py

```
@@ -1,390 +1,176 @@
-
-        if msg_type not in SUPPORTED_MESSAGE_TYPES:
-            return False
-
-        validators = {
-            "sync": self._validate_sync_payload,
-            "tx": self._validate_transaction_payload,
-            "block": self._validate_block_payload,
-        }
-        return validators[msg_type](payload)
-
     def _message_id(self, msg_type, payload):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_message_id`

(ANN202)